modules/manage/pages/manage-logging/manage-logging.adoc

```diff
@@ -444,3 +446,54 @@ The Logging REST API provides the endpoints for retrieving log and diagnostic in
 
 To retrieve log information use the `/diag` and `/sasl_logs`
 xref:rest-api:logs-rest-api.adoc[REST endpoints].
+
+[#adjust-threshold-slow-op-logging]
+== Adjusting Threshold for Logging Slow Operations
+
+It is possible to examine and/or alter the logging threshold for slow-running operations.
+This is done using the `mcctl` command that comes packaged with the Couchbase server installation.
```

Comment:
I believe this would need to be done for each node of the cluster, but it would be good to check with primary sources.  If so, it would be good to mention that here.

Also, are the settings persisted?  I honestly don't know, but a user would want to know.

Reply:
Yes, that makes sense. I'll have a chat with someone and confirm

Reply:
The settings are not persisted between restarts.

Reply:
Yes, this does need to be done for each node in the cluster. I'll add a note.